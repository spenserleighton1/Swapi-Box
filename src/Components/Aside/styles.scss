.aside {
  width: 350px;
  border: 2px solid red;
}