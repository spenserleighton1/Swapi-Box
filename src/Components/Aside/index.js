import React from 'react'
import './styles.css'

const Aside = ({ movie }) => {
  // console.log(movie)
  return (
    <div className="aside">
      <p>{ movie.scroll }</p>
      <p>{ movie.title }</p>
      <p>{ movie.year }</p>
    </div>
    )
}

export default Aside;