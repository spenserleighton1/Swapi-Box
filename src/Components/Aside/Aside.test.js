import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme'
import Aside from './';

describe('Aside', () => {
  let wrapper;
  let mockMovie;

  beforeEach(() => {
    mockMovie = { 
      title: 'Star Wars',
      scroll: 'Something about Star Wars',
      year: 1983
    }
    wrapper = shallow(<Aside movie={mockMovie}/>)
  })

  it('matches the snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  });

})