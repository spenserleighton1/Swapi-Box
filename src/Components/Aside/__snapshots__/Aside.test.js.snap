// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Aside matches the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Aside
    movie={
        Object {
            "scroll": "Something about Star Wars",
            "title": "Star Wars",
            "year": 1983,
          }
    }
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <p>
          Something about Star Wars
</p>,
        <p>
          Star Wars
</p>,
        <p>
          1983
</p>,
      ],
      "className": "aside",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Something about Star Wars",
        },
        "ref": null,
        "rendered": "Something about Star Wars",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Star Wars",
        },
        "ref": null,
        "rendered": "Star Wars",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": 1983,
        },
        "ref": null,
        "rendered": 1983,
        "type": "p",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <p>
            Something about Star Wars
</p>,
          <p>
            Star Wars
</p>,
          <p>
            1983
</p>,
        ],
        "className": "aside",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Something about Star Wars",
          },
          "ref": null,
          "rendered": "Something about Star Wars",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Star Wars",
          },
          "ref": null,
          "rendered": "Star Wars",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": 1983,
          },
          "ref": null,
          "rendered": 1983,
          "type": "p",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
